### DISCUSSION TITLE
FrozenTrial

### DISCUSSION THREAD

**Discussion by prativapokhrel** [optuna/optuna#4356] [Q&A]
### Don't use GitHub Issues to ask support questions.

![image](https://user-images.githubusercontent.com/20887144/213516780-b0ba62b4-c68a-47a1-b9d2-2d7858927668.png)

I am new to Optuna and I got this FrozenTrial at the end of my trial. Could someone please explain what this means? I could not find good documentation explaining this.  
Thank you!

**Comment by nzw0301** [comment]
Did you read https://optuna.readthedocs.io/en/stable/reference/generated/optuna.trial.FrozenTrial.html?
